orbit/cmd/desktop/desktop.go

```diff
@@ -4,43 +4,47 @@ import (
 	_ "embed"
 	"errors"
 	"fmt"
-	"log"
 	"net/url"
 	"os"
 	"path"
+	"path/filepath"
+	"runtime"
 	"time"
 
 	"github.com/fleetdm/fleet/v4/pkg/open"
 	"github.com/fleetdm/fleet/v4/server/service"
 	"github.com/getlantern/systray"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"gopkg.in/natefinch/lumberjack.v2"
 )
 
 var version = "unknown"
 
 func main() {
+	setupLogs()
+
 	// Our TUF provided targets must support launching with "--help".
 	if len(os.Args) > 1 && os.Args[1] == "--help" {
 		fmt.Println("Fleet Desktop application executable")
 		return
 	}
-	log.Printf("fleet-desktop version=%s\n", version)
+	log.Info().Msgf("fleet-desktop version=%s\n", version)
```

Comment:
Nit, new line is not required anymore (`log` packages generally append-it/take-care-of-it).